packages/blade/src/components/Modal/Modal.web.tsx

```diff
@@ -1,8 +1,9 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import React, { useEffect } from 'react';
-import styled, { css, keyframes } from 'styled-components';
+import styled, { css } from 'styled-components';
 import { FloatingFocusManager, FloatingPortal, useFloating } from '@floating-ui/react';
 import usePresence from 'use-presence';
+import { m } from 'framer-motion';
```

Comment:
nice

Reply:
Our modal seems to lag a bit too much now with this though. Its a bit too visible 😭 . Lets go back to previous CSS animation and lets live with that flash of content instead?

https://github.com/user-attachments/assets/b9884a77-5dfa-4e41-bfc4-fb8e08cc21a0

Reply:
@saurabhdaware reverted changes